src/pages/blog/blogDetails/helperFunction.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Parses html string and finds unclosed empty or void elements, and closes them.
+ * @param htmlString
+ * @returns {string}
+ */
+
+export const searchForEmptyElement = (htmlString) => {
+    const emptyElements = [
+        "<area", "<base", "<br", "<col", "<embed", "<hr", "<img", "<input", "<keygen", "<link", "<meta", "<param", "<source", "<track", "<wbr"
+    ]
+    let str = htmlString  // string to evaluate
```

Comment:
I'm not convince this variable is necessary. You should be able to just replace all instances of it with `htmlString`